scripts/package_frontend.ps1

```diff
@@ -10,4 +10,4 @@ cp backend dist -r -Force
 # Node
 cd frontend
 npm install
-npm run build
+VITE_ENV_DIR=$(dirname $(azd env list --output json | jq -r '.[] | select(.IsDefault == true) | .DotEnvPath')) npm run build
```

Comment:
I'm not sure how to test this. @ross-p-smith did you actually deploy from windows (not WSL) to test this flow with the powershell script and deployment?

Reply:
Yes I set everything up in Windows and tested. Why does the build need to know where the env file is? Isnt that a runtime concern?

Reply:
But "why" do we need environment variables at build time? The values can change at runtime, so I don't get it. Having variables in a file is fine for local debugging etc, but I would want to be able to override them at run time, otherwise I lose the benefit of having it configurable.

Reply:
The frontend React app running in the background cannot read environment variables, I think we have to set them at build time, or fetch them from an API call

You could maybe piggy-back off this PR: #680